server/routerlicious/packages/services/src/redis.ts

```diff
@@ -14,50 +14,64 @@ import { Lumberjack } from "@fluidframework/server-services-telemetry";
  */
 export class RedisCache implements ICache {
 	private readonly expireAfterSeconds: number = 60 * 60 * 24;
-	private readonly prefix: string = "page";
+	private readonly prefix: string;
+
 	constructor(private readonly client: Redis.default, parameters?: IRedisParameters) {
 		if (parameters?.expireAfterSeconds) {
 			this.expireAfterSeconds = parameters.expireAfterSeconds;
 		}
 
 		if (parameters?.prefix) {
 			this.prefix = parameters.prefix;
+		} else {
+			this.prefix = "";
+			winston.warn("A prefix for RedisCache was not included in the parameters.");
```

Comment:
Not needed. Actually, it's a recommended practice to remove all winston code when we touch a file. :)